manifests/profile/www_lib/php.pp

```diff
@@ -76,18 +76,17 @@
     },
   }
 
-  # TODO: migrate this to fpm config
-  #  class { 'php::apache_config':
-  #
-  #    settings     => {
-  #      'PHP/short_open_tag'          => 'On',
-  #      'PHP/max_input_vars'          => '2000',
-  #      'PHP/memory_limit'            => '256M',
-  #      'PHP/error_reporting'         => 'E_ALL & ~E_DEPRECATED',
-  #      'PHP/upload_max_filesize'     => '32M',
-  #      'Date/date.timezone'          => 'America/Detroit',
-  #      'mail function/sendmail_path' => '/usr/sbin/sendmail -t -i'
-  #    },
-  #  }
+  php::config { 'fpm php.ini':
```

Comment:
Since this is specific to 7.3, we should move it to that profile. I don't see anything here that is www-lib specific.

Reply:
Looked at other places where php73 is included and it doesn't look like this should be a problem. Done.